app/javascript/controllers/autocompleter_controller.js

```diff
@@ -1832,8 +1832,10 @@ export default class extends Controller {
     this.fetch_request = null;
     // Record string actually used to do matching: might be less strict
     // than one sent in request.
-    if (new_primer.length > 0)
-      this.last_fetch_request = new_primer[0]['name'];
+
+    // Why the following code?  The last_fetch_request is already correct.
+    // if (new_primer.length > 0)
+    //   this.last_fetch_request = new_primer[0]['name'];
```

Comment:
The intention of the commented-out line seems to be handling the case when the user is deleting letters from their string. 
Like if they start typing "pohylo..." and then they delete back to "p" in order to retype "phylo" correctly.

Just need to test manually that this works without the line.

Reply:
Cleaned up comments.

Reply:
Manually seems to work great. Thanks for doing this.